cyrw/bus_pico.go

```diff
@@ -1,4 +1,4 @@
-//go:build pico
+//go:build pico && cy43nopio
```

Comment:
Maybe use !cy43pio?

Reply:
We'd want pio to be default implementation used when no build tag to specify is used, which means the build tag we add has to be a build tag that uses the non-pio version, hence the `nopio`

Reply:
Gotcha, that makes sense.